swim/node.go

```diff
@@ -243,6 +265,7 @@ func (n *Node) RegisterListener(l EventListener) {
 func (n *Node) Start() {
 	n.gossip.Start()
 	n.suspicion.Reenable()
+	n.healer.Start()
```

Comment:
Like we discovered, don't forget to actually turn it on. :-) And maybe document this method: when is it expected to be called, and what it really does. You remember my confusion this morning. :-)